fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsNodeInfo.java

```diff
@@ -70,6 +70,9 @@ public EsNodeInfo(String id, Map<String, Object> map, boolean httpSslEnabled) {
         if (httpMap != null) {
             String address = (String) httpMap.get("publish_address");
             if (address != null) {
+                if (address.contains("/")) {
```

Comment:
maybe we can split this address by `/` directly,

Reply:
yes, we don't need to use contains, use one-liner code.  i modified it.